### PR TITLE
Eslint fixed + UI for listing device 

### PR DESCRIPTION
**Pull request by Ranjith-K-07**
Implemented UI for showing device list

### CODE REVIEW

**Comment by teamlead-rdl** [comment]
Since we have not yet received a confirmation on the Dashboard let's merge this code with other version.

File: src/components/DashboardComponent.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import React, { useEffect, useState } from 'react';
 import './dashboard/dragResize.scss';
 import GridLayout from 'react-grid-layout';
-import { Button } from '@mui/material';
+import { Button, Fab } from '@mui/material';
```
Comment (teamlead-rdl) [inline src/components/DashboardComponent.js:4]: Can remove this as well?

File: src/components/DashboardComponent.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 import Widget from './widget/Widget';
 import Featured from './featured/Featured';
 import BarChart from './chart/barChart/BarChart';
 import LineChart from './chart/lineChart/LineChart';
+import AddIcon from '@mui/icons-material/Add';
```
Comment (teamlead-rdl) [inline src/components/DashboardComponent.js:9]: This one as well

File: src/components/user/UserLog/UserLogForm.js [hunk 3]
```diff
@@ -46,20 +47,25 @@ export default function UserLogForm() {
...
+      field: 'createdDate',
+      headerName: 'Date',
+      width: 250,
+    },
+    {
+      field: 'createdTime',
+      headerName: 'Time',
+      width: 250,
     },
     // {
```
Comment (teamlead-rdl) [inline src/components/user/UserLog/UserLogForm.js:67]: You can remove this commented code

File: src/components/DashboardComponent.js (hunk 1 above)
Comment (Ranjith-K-07) [reply src/components/DashboardComponent.js:4]: Let it be there only right now. Because later stage we are going to use that.

**Review by teamlead-rdl — APPROVED** [review]

**Pull request merged**